examples/birds.py

```diff
@@ -0,0 +1,126 @@
+from mantisshrimp.all import *
+import scipy.io
```

Comment:
Does this add extra scipy dependency. I guess matplotlib already has that.

Reply:
This is only inside examples though =)